file-attribute-auditor/file-attribute-auditor.ipynb

```diff
@@ -0,0 +1,1717 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "eb03a932",
+      "metadata": {
+        "id": "eb03a932"
+      },
+      "source": [
+        "# Import dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "d1145a0d-338e-4a9a-984a-38c7c77661f9",
+      "metadata": {
+        "id": "d1145a0d-338e-4a9a-984a-38c7c77661f9"
+      },
+      "outputs": [],
+      "source": [
+        "import json, requests\n",
+        "import pandas as pd\n",
+        "from pandas import json_normalize\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "eZruwheZoG9V",
+      "metadata": {
+        "id": "eZruwheZoG9V"
+      },
+      "source": [
+        "# Configure Connection Variables"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "dyKrOfu_oGsy",
+      "metadata": {
+        "id": "dyKrOfu_oGsy"
+      },
+      "outputs": [],
+      "source": [
+        "# Example hostnames: \"platform.tetrascience.com\" \"tetrascience-uat.com\"\n",
+        "tdpHostname = \"tetrascience-uat.com\"\n",
+        "orgSlug = \"project-amplify\"\n",
```

Comment:
Generally we leave it blank, I think there's a different error message like "orgSlug is required" as opposed to just sending an invalid value.
```suggestion
        "orgSlug = \"\"\n",
```